unfoldingWord/create-app
# Added Resource Workspace RCL

## Test Instructions

- Open the Netlify Deploy preview on https://deploy-preview-19--create-app.netlify.app/
- The app should render three cards with tn, ta, and tw.
- Cards should be resizable and draggable 


This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword/create-app/19)
